### DISCUSSION TITLE
Any way to dump Pairing Data in WEBLOG ?

### DISCUSSION THREAD

**Discussion by erickfr06** [HomeSpan/HomeSpan#879] [Q&A] [answered]
Hi everybody.

I'm looking for a simple way to dump Pairing Data (for cloning in case of problem), but with an output directed to WEBLOG instead of Serial Console (P CLI command equivalent). This is necessary when some devices are no more physically accessibles, and pairing was done at their final locations, without possible serial connection.

For such remote locations, recovery in case of dead ESP32, without impact on home app and logics, requires to have read pairing information after installation via WiFi, and this is not possible yet.

Something like ControllerListBegin()/End() introduced in 1.9.1 but for Pairing Data should be very usefull ...

Thanks for your suggestions.

**Comment by HomeSpan** [comment, marked as answer]
That's a good suggestion. Let me give some thought and see about adding something in the next release (will not be for a while since I just released 1.9.1 yesterday).

**Reply by erickfr06** [reply]
Thanks a lot !
